### PR TITLE
Update speedtest tiers and reward multiplier based on hip-98

### PR DESCRIPTION
**Pull request by bbalser**

### CODE REVIEW

File: mobile_verifier/src/speedtests_average.rs [hunk 1]
```diff
@@ -180,6 +186,8 @@ impl SpeedtestTier {
 
     fn from_latency(latency: u32) -> Self {
```
Comment (mawdegroot) [inline mobile_verifier/src/speedtests_average.rs:187]: [HIP98](https://github.com/helium/HIP/blob/afd363bd261ae13d4f732dba7b62f09f2d119516/0098-mobile-subdao-quality-of-service-requirements.md) states:

| Speed Test Tier | Speedtest Multiplier | Requirement (speeds in Mbps, latency in ms)    |
| --------------- | -------------------- | ---------------------------------------------- |
| Good            | 1.00X                | 100+ Download, AND 10+ Upload, AND <50 Latency |
| Acceptable      | 0.75X                | 75+ Download, AND 8+ Upload, AND <60 Latency   |
| Degraded        | 0.50X                | 50+ Download, AND 5+ Upload, AND <75 Latency   |
| Poor            | 0.25X                | 30+ Download, AND 2+ Upload, AND <100 Latency  |
| Fail            | 0.00X                | <30 Download, OR <2 Upload, OR >100 Latency    |

Therefore the `<=`'s in the latency check should be changed to `<` to match the HIP text.

**Review by mawdegroot — COMMENTED** [review]
The latency check does not the limits correspond to the HIP text.

File: mobile_verifier/src/speedtests_average.rs (hunk 1 above)
Comment (bbalser) [reply mobile_verifier/src/speedtests_average.rs:187]: Good eyes, fixed febdcce

File: mobile_verifier/src/speedtests_average.rs [hunk 2]
```diff
@@ -295,38 +303,39 @@ mod test {
...
+            SpeedtestTier::Good.min(SpeedtestTier::Failed),
             SpeedtestTier::Failed,
         );
     }
 
     #[test]
     fn check_known_valid() {
         let speedtests = known_speedtests();
-        assert_ne!(
+        dbg!(SpeedtestAverage::from(speedtests[0..5].to_vec()).tier());
```
Comment (maplant) [inline mobile_verifier/src/speedtests_average.rs:314]: Remove this debug from the test.

**Pull request merged**